control-plane/plugin/src/resources/volume.rs

```diff
@@ -71,7 +71,8 @@ impl CreateRow for openapi::models::Volume {
                     VolumeContentSource::snapshot(_) => "Snapshot",
                 }
             })),
-            optional_cell(state.health.as_ref().map(|h| h.clean_shutdown))
+            optional_cell(state.health.as_ref().map(|h| h.clean_shutdown)),
+            self.spec.encrypted
```

Comment:
Yeah see above the snapped thing,
so maybe here we could do something similar:
```suggestion
            match self.spec.encrypted {
                false => "false",
                true if all_healthy_replicas_encrypted => "true",
                true => "true (requires rebuild)",
},
```
or

```suggestion
            match self.spec.encrypted {
                false => "disabled",
                true if all_healthy_replicas_encrypted => "enabled (secured)",
                true => "enabled (rebuild required)",
},
```

Reply:
Sure. I will take this up on a follow up PR. For now keeping it the way it is.